README.md

```diff
@@ -57,6 +56,7 @@ Ready-to-use prompt templates for specific development scenarios and tasks, defi
 - [Multi Stage Dockerfile](prompts/multi-stage-dockerfile.prompt.md) - Create optimized multi-stage Dockerfiles for any language or framework
 - [My Issues](prompts/my-issues.prompt.md) - List my issues in the current repository
 - [My Pull Requests](prompts/my-pull-requests.prompt.md) - List my pull requests in the current repository
+- [Next.js i18n with next-intl](prompts/next-intl-add-language.prompt.md) - Add a new language to a Next.js project using [next-intl](https://next-intl.dev/).
```

Comment:
Ensure that this information comes from running `update-readme.js`.

Reply:
Done (missed this)